xcode/Subconscious/Shared/Services/Noosphere/NoosphereService.swift

```diff
@@ -140,14 +140,18 @@ final class NoosphereService: SphereProtocol, SphereIdentityProtocol {
         guard let identity = self._sphereIdentity else {
             throw NoosphereServiceError.defaultSphereNotFound
         }
+        
+        return try sphere(identity: identity)
+    }
+    
+    /// Open a sphere using its identity
+    func sphere(identity: String) throws -> Sphere {
```

Comment:
Why do we need this method when we have traverse?

If we do need this method, I feel we should consider making it a synonym of `traverse`, or pursue one of the other directions below.

```swift
func traverse(identity: String) throws -> Sphere
```

Rationale: `.sphere()` is meant to return a fallible memoized instance of the _user's_ sphere — it's essentially like a property, but allowing for throwing failure.

Adding a synonym to return someone else's sphere by identity confuses the concept.

Approaches to consider:

- Synonym for `traverse`
- Move to `Noosphere` as `Noosphere.sphere(identity:)`
    - Noosphere is lower-level, not a service, so doesn't have the concept of "user's sphere" to worry about. Probably a better place for this.

Reply:
We don't actually need it at all, I was just thinking forward to code that we don't need to write yet. 

In the event we truly require a method like this one I quite like `Noosphere.sphere(identity:)` as a location.